src/core/Akka/Actor/RepointableActorRef.cs

```diff
@@ -342,7 +342,7 @@ public bool TryGetChildStatsByName(string name, out IChildStats child)
             return false;
         }
 
-        public void Post(IActorRef sender, object message)
+        public void SendMessage(IActorRef sender, object message)
```

Comment:
Going to skip that method for now - requires some yak-shaving in how we implement message sending and I need to spend time working on other things. Not essential at the moment.